portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,82 @@
+                 computer generated AI enemies run towards the player and if one touched the player,
+                 one life would be lost. We ended up winning the grand prize with this project.
+                 This was the first hackathon we ever won, so this project is near and dear to my
+                 heart. </li>
+
+            <li> "Naturally U"- This project is something I'm proud of because it's all about natural hair,
+                  something that I love about myself and know a lot about. Naturally U is a all-in-one resource for 
+                  natural hair.It shows you affordable hair products for your hair type, style tutorials, and 
+                  the closest barbers or hair stylists to the user. Naturally U was made during the UNCF CS Summer
+                  Academy where I worked with a team on 3 to complete this project using HTML,CSS,and Python.My specific role was the full 
+                  stack developer. I worked on the front end of several pages of the app including the home page and the tutorial selection
+                  page and on the back end of the login page. At the end of the summer pitch
+                  competition our project took the grand prize!</li>
+
+            <li> "Little Green Foot"- This project is a sort of environmental effect tracker. In this app
+                  you could mark "environmentally friendly" actions you did that day such as carpooling to work, recycling, etc.
+                  and you would earn points from those actions which the user could cash out those points in the
+                  form of gift cards and other prizes. My role on this project was the full stack developer. I worked 
+                  with the front end developer to create all the visuals for the mock up and I worked with the backend developer
+                  to create the point system algorithm.</l1>
+        </ul>
+        <h2> If You Could Travel Anywhere in the World,Where Would You Go Kayla? </h2>
+            <p> Wow,thank y'all for asking!</p>
+
+            <p> I would say Italy is my number 1, I've always wanted to go there but if I
+                had to choose somewhere else...</p>
+            <img src="/images/italy.jpg" alt=" a cool and free pic of Italy!" style="width:50% ;height:50%">
+
+            <p>My number 2 would be Jamaica! Its so beautiful and I've never been to a place with a
+               tropical climate before so it would be a first but if I cant choose this either my
+               resulting choice would be...</p>
+            <img src="/images/jamaica.jpg" alt=" a cool and free pic of Jamaica!" style="width:50% ;height:50%">
+
+            <p> My number 3 would be France! Can't go wrong with France, it's another beautiful
+                place with alot to do.</p>
+            <img src="/images/france.jpg" alt=" a cool and free pic of France!" style="width:50% ;height:50%">
```

Comment:
nit: if you want these images centered, you could add: `display: block; margin: 0 auto;` to the style string.